### DISCUSSION TITLE
为什么我跨链转币三个月还不到目前显示成功我还是查不到？

### DISCUSSION THREAD

**Discussion by daqiaqi** [blockscout/blockscout#9613] [Q&A]
为什么我跨链转币三个月还不到目前显示成功我还是查不到？
![Y8 (AK4T IBJ92 78MGHRN4](https://github.com/blockscout/blockscout/assets/161859852/51bc7742-0556-4eee-8b1d-017366f50797)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
